srcpkgs/webkit2gtk/template

```diff
@@ -183,11 +184,8 @@ webkit2gtk-devel_package() {
 }
 
 webkit2gtk-common_package() {
-	short_desc="GTK port of the WebKit2 browser engine - common files"
-	pkg_install() {
-		vmove usr/share/locale/
-		vmove usr/share/licenses/
-	}
+	short_desc="GTK port of the WebKit2 browser engine - (transitional dummy package)"
+	build_style=meta
 }
```

Comment:
This package installed locale files to /usr/share/locale/locale, so it won't conflict with the main package and I think we can just remove it instead of creating a dummy package.

Reply:
It grabs the vlicense installed by webkit2gtk and moved it to webkit2gtk-common, which will conflict with webkit2gtk. Is that an issue?

Reply:
It seems like they are also in the wrong directory, make sure they will now be in the correct directory and it should be fine. Also, $replaces in webkit2gtk should uninstall webkit2gtk-common, no?
```
%: xls webkit2gtk-common | grep -v locale
/usr/share/licenses/licenses/webkit2gtk/LICENSE-APPLE
/usr/share/licenses/licenses/webkit2gtk/LICENSE-LGPL-2
/usr/share/licenses/licenses/webkit2gtk/LICENSE-LGPL-2.1
```

Reply:
Oh, you're right. Lol.